test.py

```diff
@@ -1141,6 +1143,17 @@ def parse_cmd_line() -> argparse.Namespace:
                              "is only supported by python tests for now, other tests ignore it. "
                              "By default, the marker filter is not applied and all tests will be run without exception."
                              "To exclude e.g. slow tests you can write --markers 'not slow'.")
+    parser.add_argument('--kexpr', action='store', metavar='EXPRESSION',
```

Comment:
Need a long (full) option name. Besides, there is pattern matching for command line, what happens if both is given?

Reply:
`test.py` 'name' picks the suite/pytest, and '--kexpr' is passed on `pytest` command line. It's a 2 level filter as it is now (in the original I wanted to use it in 'name' but was asked to do it this way).